webapp/javascript/components/TimelineChart/TimelineChartWrapper.tsx

```diff
@@ -71,6 +71,7 @@ type TimelineChartWrapperProps = TimelineDataProps & {
 
   /** selection type 'single' => gray selection, 'double' => color selection */
   selectionType: 'single' | 'double';
+  exploreTooltip?: FC<ShamefulAny>;
```

Comment:
Can we type it better? Either `unknown` if its props are not really used, or to the real thing?

Reply:
fixed